Examples/Tests/initial_distribution/analysis_distribution.py

```diff
@@ -17,6 +17,11 @@
 import scipy.constants as scc
 import scipy.special as scs
 from read_raw_data import read_reduced_diags_histogram, read_reduced_diags
+import sys
+sys.path.insert(1, '../../../../warpx/Regression/Checksum/')
+import checksumAPI
+
+filename = sys.argv[1]
```

Comment:
It looks like the checksum were not tested in the analysis script, so I added them.

Reply:
So the benchmarks were failing but it seems to be unrelated to this PR (I tried reverting the changes of the PR, except for these lines, which activate the checksum test in the analysis script, and got the same thing).

Most notably, the electric field checksums are now higher by a factor 100 (almost exactly) and the magnetic field checksums are now higher by a factor 10000 (almost exactly). Should we worry about this? (for now I've updated the benchmark but this can be reverted)

@MaxThevenet Do you know if there is a reason why the checksum were not tested in this analysis script?

Reply:
Hm, `WarpX-tests.ini` might need to add to `runtime_params =` the options `warpx.do_dynamic_scheduling=0 warpx.serialize_ics=1` for this test (`[initial_distribution]`)?

Reply:
Good idea, I've added that in the latest commit. It didn't change the benchmarks though (it would have been surprising if this changed the magnetic field by 4 orders of magnitude).

The probably is probably due to something that changed in the code since those benchmark values were added. Do you think it's worth tracking it down?

Reply:
I think it's fine - the B-field is essentially close to zero and we changed the microscopic random number generator in the PR. LGTM, this test is more about the init of the particles which looks good.